hera_mc/tests/test_qm.py

```diff
@@ -183,11 +180,11 @@ def test_update_qm_list(self):
         r = self.test_session.get_metric_desc()
         results = []
         for result in r:
-            self.assertTrue(result.metric in self.metrics_dict.keys())
+            self.assertTrue(result.metric in list(self.metrics_dict.keys()))
             results.append(result.metric)
-        for metric in self.metrics_dict.keys():
+        for metric in list(self.metrics_dict.keys()):
```

Comment:
Since all that's being done with the `keys()` value is iterating over it, the listification isn't really necessary here.

Reply:
thanks!